golang/cosmos/x/swingset/types/store.go

```diff
@@ -0,0 +1,141 @@
+package types
+
+import (
+	"bytes"
+	"io"
+
+	prefixstore "github.com/cosmos/cosmos-sdk/store/prefix"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+)
+
+var (
+	SwingStoreValueSuffix = []byte{0}
+)
+
+var _ storetypes.KVStore = Store{}
+
+// Store is a KVStore, optionally prefixed, but allowing nil values
+type Store struct {
```

Comment:
Can we rename this struct to `kvStore`?
It will save use from writing the above comment?
Also, if not, then I am curious if we can lowercase to `store` to align with other struct naming convention in this PR. We might later want to introduce a linter that can help not diverge in naming. Currently half of the structs start with [A-Z] and other half [a-z] in go code.
`grep -h -R --include \*.go "type .* struct.*" | sort`

Reply:
This struct is public: it's used as the return type of

https://github.com/Agoric/agoric-sdk/blob/761d54fb85245f674810d99d01086b21ebc2c228/golang/cosmos/x/swingset/keeper/keeper.go#L440

(from what I understand, in golang, an identifier starting with lower case is considered private to the package)